pom.xml

```diff
@@ -50,6 +50,8 @@
     <gitHubRepo>jenkinsci/opentelemetry-api-plugin</gitHubRepo>
     <revision>${opentelemetry.version}</revision>
     <spotless.check.skip>false</spotless.check.skip>
+    <hpi.strictBundledArtifacts>true</hpi.strictBundledArtifacts>
+    <hpi.bundledArtifacts>opentelemetry-api,opentelemetry-api-incubator,opentelemetry-context,opentelemetry-exporter-common,opentelemetry-exporter-logging,opentelemetry-exporter-otlp,opentelemetry-exporter-otlp-common,opentelemetry-exporter-prometheus,opentelemetry-exporter-sender-okhttp,opentelemetry-instrumentation-api,opentelemetry-instrumentation-api-incubator,opentelemetry-resources,opentelemetry-sdk,opentelemetry-sdk-common,opentelemetry-sdk-extension-autoconfigure,opentelemetry-sdk-extension-autoconfigure-spi,opentelemetry-sdk-logs,opentelemetry-sdk-metrics,opentelemetry-sdk-trace,opentelemetry-semconv,opentelemetry-semconv-incubating,prometheus-metrics-config,prometheus-metrics-exporter-common,prometheus-metrics-exporter-httpserver,prometheus-metrics-exposition-formats,prometheus-metrics-exposition-textformats,prometheus-metrics-model</hpi.bundledArtifacts>
```

Comment:
This is not a good idea; managing the bundle dependencies manually is a burden I do not want to make.